lib/node/nodes/data-observatory-multiple-measures.js

```diff
@@ -49,6 +49,36 @@ var DataObservatoryMultipleMeasures = Node.create(TYPE, PARAMS, { cache: true, l
             throw new Error('The number of numerators=' + this.numerators.length +
                 ' does not match the number of numerator_timespans=' + this.numerator_timespans.length);
         }
+
+        this.columnTypes = this.numerators.reduce((columnTypes, numeratorId, index) => {
+            columnTypes[numeratorId] = {
+                name: this.column_names[index],
+                type: 'Numeric'
+            };
+            return columnTypes;
+        }, {});
+    },
+    beforeCreateAsync: function(databaseService, callback) {
+        const sql = columnTypesQueryTemplate({
+            source: this.source.getQuery(false),
+            obsMetaParams: this._composeObsMetaParams()
+        });
+        databaseService.run(sql, (err, result) => {
```

Comment:
Short answer: YES! 😄 

I started to use it for the templates, where it is quite handy, so I continued the usage for the rest of the code.